tools/build_executable.py

```diff
@@ -0,0 +1,108 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--unreal_engine_dir", required=True)
+    parser.add_argument("--conda_env", default="spear-env")
+    parser.add_argument("--config_mode", default="Shipping")
+    parser.add_argument("--num_parallel_jobs", type=int, default=1)
+    parser.add_argument("--output_dir", default=os.path.dirname(os.path.realpath(__file__)))
+    parser.add_argument("--temp_dir", default=os.path.join(os.path.dirname(os.path.realpath(__file__)), "tmp"))
+    parser.add_argument("--conda_script")
+    parser.add_argument("--commit_id")
+    args = parser.parse_args()
+
+    repo_dir           = os.path.join(os.path.realpath(args.temp_dir), "spear")
+    unreal_project_dir = os.path.realpath(os.path.join(repo_dir, "cpp", "unreal_projects", "SpearSim"))
+    unreal_plugins_dir = os.path.realpath(os.path.join(repo_dir, "cpp", "unreal_plugins"))
+
+    # remove any contents already present for a clean build
+    if os.path.exists(repo_dir):
+        shutil.rmtree(repo_dir)
+        os.makedirs(repo_dir)
+
+    if sys.platform == "win32":
+        run_uat_script = os.path.join(args.unreal_engine_dir, "Engine", "Build", "BatchFiles", "RunUAT.bat")
+        target_platform = "Win64"
+        archive_dir = os.path.join(args.output_dir, f"SpearSim-{target_platform}-{args.config_mode}")
+        cmd_prefix = f"conda activate {args.conda_env}&"
+    elif sys.platform == "darwin":
+        run_uat_script = os.path.join(args.unreal_engine_dir, "Engine", "Build", "BatchFiles", "RunUAT.sh")
+        target_platform = "Mac"
+        archive_dir = os.path.join(args.output_dir, f"SpearSim-{target_platform}-{args.config_mode}-Unsigned")
+        conda_script = args.conda_script
+        if not conda_script:
+            conda_script = "~/anaconda3/etc/profile.d/conda.sh"
```

Comment:
`os.path.join( "~", "anaconda3", "etc", "profile.d", "conda.sh")`

Strictly speaking, you don't need to do this, but it's just for consistency with `run_uat_script`.

Reply:
Done